### ISSUE TITLE
Bug in to_ref method of Bill

### ISSUE THREAD

**Issue by subodh-malgonde** [sidecars/python-quickbooks#64]
This is the current code of `to_ref` method:

```
def to_ref(self):
        ref = Ref()

        ref.name = self.DisplayName
        ref.type = self.qbo_object_name
        ref.value = self.Id

        return ref
```

Bill objects do not have a property `DisplayName`.

**Comment by ab10460ef3** [comment]
DocNumber for DisplayName

```
    def to_ref(self):
        ref = Ref()

        ref.name = self.DocNumber
        ref.type = self.qbo_object_name
        ref.value = self.Id

        return ref
```

**Comment by ej2** [comment]
Fixed in release 0.6.1
